src/nrnoc/init.cpp

```diff
@@ -1109,11 +1109,3 @@ double nrn_call_mech_func(Symbol* s, int narg, Prop* p, int type) {
     }
     return hoc_call_func(s, narg);
 }
-
-void nrnunit_use_legacy() {
-    if (ifarg(1)) {
-        int arg = (int) chkarg(1, 0, 1);
-        _nrnunit_use_legacy_ = arg;
-    }
-    hoc_retpushx((double) _nrnunit_use_legacy_);
-}
```

Comment:
After discussing this in the NEURON Hackathon it was proposed that it would be nice to get an error if the legacy units are selected saying that the legacy units are no longer supported and the user can change manually the different values in `nrnunits.lib`

Reply:
ok

Reply:
Done